model/src/main/java/com/ge/predix/acs/model/Attribute.java

```diff
@@ -41,7 +41,7 @@
     private List<Attribute> scopes;
 
     public Attribute() {
```

Comment:
scopes need to be removed from this class ?    Also ScopedAttribute class is not used, I think - the class and its references need to be removed.